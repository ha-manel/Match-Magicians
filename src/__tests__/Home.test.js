import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/home';

describe('Test of home page', () => {
  it('test if home page is rendered', () => {
    const homePage = renderer
      .create(<Home />)
      .toJSON();
    expect(homePage).toMatchSnapshot();
  });
});
