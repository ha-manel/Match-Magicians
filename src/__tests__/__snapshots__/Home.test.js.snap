// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of home page test if home page is rendered 1`] = `
<div
  id="home-page"
>
  <h2
    id="home-header"
  >
    Welcome to our page
  </h2>
  <p
    id="home-text"
  >
    Mathematics is the science that deals with the logic of shape, quantity and arrangement. Math is all around us, in everything we do. It is the building block for everything in our daily lives, including mobile devices, computers, software, architecture (ancient and modern), art, money, engineering and even sports.
  </p>
</div>
`;
